### DISCUSSION TITLE
How to display read name in the AlignmentsTrack?

### DISCUSSION THREAD

**Discussion by tao-bioinfo** [GMOD/jbrowse-components#3541] [Q&A]
Now I can only hover the mouse pointer to the read to see its name.

In my track there are only very few reads and I hope to display the read names by default. How should I do? Thank you!

**Comment by cmdcolin** [comment]
there is no functionality in the alignments renderer to add labels to the features so unfortunately can't do this currently. might need to e.g. convert to a BED file for this. if you have any more info about your use case, might be able to add also :) good to here from you @billzt

**Reply by cmdcolin** [reply]
and, just to be clear, converting to a BED file would not draw the alignments subfeatures like mismatches,inserations,deletions...just would be plain boxes but would have labels

**Reply by tao-bioinfo** [reply]
Yes. In fact I want to see the mismatches between multiple primers and templates. Showing them in a AlignmentsTrack seems good.

![image](https://user-images.githubusercontent.com/1146090/221327658-95570a7a-843a-4e6d-a0a4-64504bcc8d4c.jpeg)
